docs/API.md

```diff
@@ -170,7 +170,7 @@ Basic data type: object
 Properties:
 
 * `"ns"`: A *domain name*, required.
-* `"ip"`: An *IP address*, IPv4 or IPv6 , optional.
+* `"ip"`: An *IP address* (IPv4 or IPv6) or null, optional. (default: `null`)
```

Comment:
The implementation needs to be updated to actually allow `null` here. This is probably true for most other `null` default values as well.

Reply:
Or just not allow "ip" with `null` value.

Reply:
i agree with mats.

Reply:
In that case we need to come up with another allowed default value.

Reply:
I just send a new commit :)
No default value